prisma/dev-seeds/index.ts

```diff
@@ -0,0 +1,12 @@
+import classUpsert from "./class";
+import programUpsert from "./program";
+import programTranslationsUpsert from "./program-translation";
+
+/**
+ * Seed the Development environment
+ */
+export default async function seedDev(): Promise<void> {
```

Comment:
I think there might be a need to seed a teacher user as well. Can you check the current staging database to see if we have a teacher user. If there is one, I believe we need it as I think we use the name field of the teacher. @MatoPlus

Reply:
You're absolutely right 👍 . I was working the page to do parent class registration and the model needs some more sample data in `Teacher` along with `ClassTranslation`. I'll try to add those data along with anything else that may seem important later today.